packages/core/review-workflows/admin/src/routes/settings/components/Stages.tsx

```diff
@@ -82,7 +82,7 @@ const Stages = ({ canDelete = true, canUpdate = true, isCreating }: StagesProps)
                   canReorder={stages.length > 1}
                   canUpdate={canUpdate}
                   stagesCount={stages.length}
-                  defaultOpen={isCreating}
+                  defaultOpen={!stage.id}
```

Comment:
Can you explain this change?

Reply:
I was expecting that every time you click at "Add new stage" the accordion is opened in the new item. That is what I thought when using it because I don't see many sense on opening when creating a new stage **only** when is a new workflow but if you are editing and you add a new stage is not opened. (This also simplified a lot the e2e test)

But I would check with Lucas about this 🤔

Reply:
That sounds great to me 👍